AK/Tests/TestWeakPtr.cpp

```diff
@@ -32,6 +32,7 @@
 class SimpleWeakable : public Weakable<SimpleWeakable> {
 public:
     SimpleWeakable() {}
+    inline void SetMember(int val) { m_member = val; }
```

Comment:
Please adhere to the project coding style:
```suggestion
    void set_member(int value) { m_member = value; }
```

Also I think you can use `[[maybe_unused]]` on the member instead of this.

Reply:
I will try! 👍

Reply:
Shame, it didn't work, I will just drop set_member there for now, if that's ok.

Reply:
Sure!